kong/db/strategies/off/init.lua

```diff
@@ -213,6 +214,15 @@ local function select_by_key(schema, key)
 
   entity = schema:process_auto_fields(entity, "select", true, PROCESS_AUTO_FIELDS_OPTS)
 
+  if entity and entity.ttl then
+    local ttl_value = entity.ttl - now()
+    if ttl_value > 0 then
+        entity.ttl = ttl_value
+    else
+        entity.ttl = 0
+    end
```

Comment:
Why do we keep the expiration time but then use it to calculate the TTL?

Reply:
TTL(Time to live) is a relative value usually, the remaining time until expiration. 

But I found we have stored the absolute value of expired timestamp into Postgre with the field name "ttl".  So here it saves the expired timestamp into lmdb to align the original logic.

```
kong=# \d keyauth_credentials
...
 ttl         | timestamp with time zone |
```

For the higher-level caller, they utilize the `cred.ttl` as the remaining time until expiration. Therefore, we need to convert the timestamp into the remaining time, and this conversion is referred to as readtime.

Reply:
The style remains consistent with the original code, such as postgres.